File: src/main/org/nlogo/window/LocalViewInterface.java
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.window;

public interface LocalViewInterface
    extends org.nlogo.api.ViewInterface {
  java.awt.Component getExportWindowFrame();

  java.awt.image.BufferedImage exportView();

  void displaySwitch(boolean on);

  boolean displaySwitch();
}

File: src/main/org/nlogo/window/LocalViewInterface.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.window

trait LocalViewInterface extends org.nlogo.api.ViewInterface {
  def getExportWindowFrame: java.awt.Component
  def exportView(): java.awt.image.BufferedImage
  def displaySwitch(on: Boolean): Unit
  def displaySwitch: Boolean
}
